lib/ansible/modules/packaging/os/zypper.py

```diff
@@ -288,6 +294,8 @@ def get_cmd(m, subcommand):
     is_refresh = subcommand == 'refresh'
     cmd = ['/usr/bin/zypper', '--quiet', '--non-interactive', '--xmlout']
 
+    if subcommand == 'install' and m.params['installroot']:
```

Comment:
What about `update` and `patch`? Do those work with `--root` as well?

Reply:
it seems update and patch   don't have this option if I understand zypper docs correct

Reply:
Since `--root` comes before the subcommand it should work on all subcommands. How would you update a package in a different installroot otherwise?